tests/test_integration.py

```diff
@@ -10,9 +11,16 @@ def test_eq_success(self):
         expect(1).to == 1
         expect(1) == 1
 
+    def test_not_eq_success(self):
+        expect(1).not_to.eq(2)
+        expect([1, 2]).not_to.eq([2, 1])
+        expect((1, 2)).not_to.eq((2, 1))
+        expect(1).not_to == 2
```

Comment:
`to_not` and `not_to` are both correct, but `not_to` is more popular, see the discussion here: https://github.com/EastAgile/robber.py/issues/13